blog.py

```diff
@@ -285,6 +306,8 @@ def _load_post(self, post):
         # parse content index to end
         content = ''.join(lines)
         empty = self._is_ess_empty(lines[index:])
+        title_compare = title.replace('-', ' ')
+        name_title_mismatch = self._is_name_title_mismatch(post, title_compare)
```

Comment:
Why don't you pass `title` (instead of `title_compare`) into the function `_is_name_title_mismatch`?

Reply:
because you pointed out that "Dash can exist in title. " Therefore, I replaced the '-' with space before do the comparison

Reply:
But you can replace dash with space inside the function `_is_name_title_mismatch`, right? In this way, you keep the main logic simple.